src/components/Comments/Comment.js

```diff
@@ -111,6 +129,61 @@ const Comment = ({
     return COLORS.balance_gray;
   };
 
+  const username = comment?.data?.anon_user_info_color_name
+    ? `Anonymous ${comment.data?.anon_user_info_emoji_name}`
+    : user?.data?.username;
+
+  const onPressDM = async () => {
+    try {
+      setLoading({...loading, loadingDm: true});
+      if (!comment?.data?.anon_user_info_color_name) {
+        const channelName = [username, profile?.myProfile?.username].join(',');
+        const selectedUser = {
+          user: {
+            name: channelName,
+            image: comment?.user?.data?.profile_pic_url || DEFAULT_PROFILE_PIC_PATH
+          }
+        };
+        const members = [comment?.user?.id, profile.myProfile.user_id];
+        await createSignChat(members, selectedUser);
+      } else {
+        await sendMessageDM(comment?.id, 'comment', 'SIGNED');
+      }
+    } catch (e) {
+      console.warn(e);
+    } finally {
+      refSheet.current.close();
+      setLoading({...loading, loadingDm: false});
+    }
+  };
+
+  const onPressDMAnon = async () => {
+    try {
+      setLoading({...loading, loadingDmAnon: true});
+      await sendMessageDM(comment?.id, 'comment', 'ANONYMOUS');
+    } catch (e) {
+      console.warn(e);
+    } finally {
+      refSheet.current.close();
+      setLoading({...loading, loadingDmAnon: false});
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `onPressDM` and `onPressDMAnon` functions to handle direct messaging is a good practice. However, the error handling could be improved. Instead of just logging the error to the console, consider providing feedback to the user about the failure.

```diff
+ const onPressDM = async () => {
+   try {
+     setLoading({...loading, loadingDm: true});
+     //...
+   } catch (e) {
+     console.warn(e);
+   } finally {
+     refSheet.current.close();
+     setLoading({...loading, loadingDm: false});
+   }
+ };
+ 
+ const onPressDMAnon = async () => {
+   try {
+     setLoading({...loading, loadingDmAnon: true});
+     //...
+   } catch (e) {
+     console.warn(e);
+   } finally {
+     refSheet.current.close();
+     setLoading({...loading, loadingDmAnon: false});
+   }
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->